src/test/java/com/github/mkopylec/sessioncouchbase/TestApplication.java

```diff
@@ -1,11 +1,13 @@
 package com.github.mkopylec.sessioncouchbase;
 
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Import;
 
 import static org.springframework.boot.SpringApplication.run;
 
 @EnableCouchbaseHttpSession
 @SpringBootApplication
+@Import(TestConfiguration.class)
```

Comment:
The `@Import` is unnecessary here. `@SpringBootApplication` scans the package for `@Configuration`s